com.gitlab.bitseater.meteo.json

```diff
@@ -0,0 +1,148 @@
+{
+    "app-id" : "com.gitlab.bitseater.meteo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.gitlab.bitseater.meteo",
+    "build-options" : {
+        "cflags" : "-O2  -Wno-deprecated-declarations",
+        "cxxflags" : "-O2",
+        "env" : {
```

Comment:
ideally clean this up

Reply:
Cleaned with @bilelmoussaoui  comments. Thanks